configuration/configuration.py

```diff
@@ -172,10 +181,6 @@ def read_secret(secret_name):
 # Time zone (default: UTC)
 TIME_ZONE = os.environ.get('TIME_ZONE', 'UTC')
 
-# The Webhook event backend is disabled by default. Set this to True to enable it. Note that this requires a Redis
-# database be configured and accessible by NetBox (see `REDIS` below).
-WEBHOOKS_ENABLED = os.environ.get('WEBHOOKS_ENABLED', 'False').lower() == 'true'
```

Comment:
Is the Webhook system not optional anymore?

Reply:
No. The worker shall in the future be used for other background tasks besides webhooks.
See https://github.com/netbox-community/netbox/releases/tag/v2.7-beta1:

> ### WEBHOOKS_ENABLED Configuration Setting Removed ([#3408](https://github.com/netbox-community/netbox/issues/3408))
> 
> As django-rq is now a required library, NetBox assumes that the RQ worker process is running. The installation and upgrade documentation has been updated to reflect this, and the WEBHOOKS_ENABLED configuration parameter is no longer used. Please ensure that both the NetBox WSGI service and the RQ worker process are running on all production installations.

Reply:
Ok, I missed that.